FeedStoreChallenge/Entities/ManagedFeedImage+CoreDataProperties.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ManagedFeedImage+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Istvan Private on 14.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+public class ManagedFeedImage: NSManagedObject {
```

Comment:
`ManagedCache` is an implementation detail of `CoreDataFeedStore`, so it should not be marked as `public`. Also, this class is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```